tools/functional.ps1

```diff
@@ -55,9 +55,17 @@ param (
 Set-StrictMode -Version 'Latest'
 $ErrorActionPreference = 'Stop'
 
+# Fix up arch and config to match build conventions.
```

Comment:
Can you put these conversions into a common file? We shouldn't have to update every file if we add a new arch mapping. (Ideally there would be no need to map between these)